configs/ruby/GPU_VIPER.py

```diff
@@ -492,6 +494,15 @@ def define_options(parser):
     parser.add_argument(
         "--glc-atomic-latency", type=int, default=1, help="GLC Atomic Latency"
     )
+    parser.add_argument(
+        "--atomic-alu-latency", type=int, default=0, help="Atomic ALU Latency"
+    )
+    parser.add_argument(
+        "--tcc-num-atomic-alus",
+        type=int,
+        default=64,
```

Comment:
Why is this default so different from the one in RubyCache.py?

Reply:
Updated RubyCache.py to match